acceptance/bundle/resources/volumes/change-name/script

```diff
@@ -14,6 +14,8 @@ title "Update name"
 trace update_file.py databricks.yml myvolume mynewvolume
 
 trace $CLI bundle plan
+# terraform marks this as "update", direct marks this as "update_with_id"
```

Comment:
Why this difference?

Reply:
The update_with_id action does the update which potentially changes object id. The update action always keeps the id.

We distinguish update_with_id from update because knowing it's "update _without id_" allows to substitute ${resources.jobs.foo.id} during plan phase.

Originally the idea was to us this to unlock more parallelism - we can deploy resources in parallel if everything is resolved during plan phase. Currently, we opt on the conservative side enforce the dependency, because that's what terraform does, but in the future we might want to enable this back.

Reply:
Is the distinction relevant to users though? It might be simpler if everything is just `"update"`?